### PR TITLE
BaSP TG-46: Employees redux create update delete

### PR DESCRIPTION
**Pull request by GuidoG95**
Used redux to add, edit and delete employees.

### CODE REVIEW

File: src/Components/Employees/index.js [hunk 1]
```diff
@@ -41,25 +41,23 @@ const Employees = () => {
...
+        });
+        setFeedbackModalDisplay(true);
+      }
     }
+    dispatch(getEmployees());
   };
 
   const employeeEdit = (item) => {
-    history.push('/employees/form', { ...item, project: item.project._id });
+    history.push('/employees/form', { ...item });
```
Comment (LucianaPrat) [inline src/Components/Employees/index.js:60]: you could change this `{ ...item } `for  `,item`

**Review by LucianaPrat — APPROVED** [review]

**Review by elias-92 — APPROVED** [review]

File: src/Components/Employees/index.js (hunk 1 above)
Comment (GuidoG95) [reply src/Components/Employees/index.js:60]: done

File: src/Components/Employees/Form/index.js [hunk 2]
```diff
@@ -24,69 +27,35 @@ const EmployeesForm = () => {
...
-        setModalDisplay(true);
-        setModalContent({ message: json.message, error: json.error });
-      });
-  };
-
   const onSubmit = (e) => {
     e.preventDefault();
     if (selectedEmployee) {
-      editItem(employeeInput);
+      console.log(employeeInput);
```
Comment (Baldizan) [inline src/Components/Employees/Form/index.js:45]: please remove this console log

Comment (GuidoG95) [reply src/Components/Employees/Form/index.js]: done

**Review by NicolasGulam — APPROVED** [review]

**Pull request merged**